lib/src/DateRangePicker/DateRangePickerInput.tsx

```diff
@@ -183,3 +184,5 @@ export const DateRangePickerInput: React.FC<DateRangeInputProps> = ({
     </div>
   );
 };
+
+DateRangePickerInput.propTypes = inputPropTypes;
```

Comment:
What do you think of duplicating the information in the propTypes? This would be one step forward how we automatically generate the proptypes from the typescript definitions in the main repository.
Is the `inputPropTypes` import pruned from the bundle?

Reply:
```
process.env.NODE_ENV !== "production" ? PureDateInput.propTypes = inputPropTypes : void 0;
```

From build code. Do you think we should duplicate it?

Reply:
I'm asking because, in the core, we are from time to time using prop-types helpers:

- exactProp
- chainPropTypes
- elementAcceptingRef
- elementTypeAcceptingRef
- refType
- deprecatedPropType
- requiredPropFactory
- unsupportedProp

It seems that none of the imports are pruned in people bundle (removing X.propTypes = Y not being enough). But to be confirmed.